backend-api/app/schemas/producers.py

```diff
@@ -5,6 +5,8 @@
 
 class ProducerBase(BaseModel):
     name: str
+    eth_address: str
+    email: str
```

Comment:
Users shouldn't be able to change their email addresses (at least not yet). The backend uses the email as their username.